migration/migration-validators.md

```diff
@@ -96,9 +96,18 @@ Once you have your client setup, you can either:
 - Go to the file `$HOME/.nimiq/client.toml`, which includes a dedicated validator section at the end, and paste your validator data into this section.
```

Comment:
I miss an additional bullet that suggests to go over all the client.toml sections and fill them in accordingly.
Similar to the one for the node operators.